e2e/test/scenarios/native/native-reproductions.cy.spec.js

```diff
@@ -248,38 +248,6 @@ describe("issue 18148", () => {
   });
 });
 
-describe("issue 18418", () => {
```

Comment:
I deleted this test, because the issue connected to it seems really old. But the main reason for deletion is that this test wasn’t testing anything. The negative assertion here would not test anything even if the described problem appeared.